src/tools/management/commands/output_user_metrics.py

```diff
@@ -0,0 +1,19 @@
+import csv
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from tools.metrics import agg_log_events
+from tools.models import LogEvent
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        les = LogEvent.objects.all()
+        agg_les = agg_log_events(les)
+        fieldnames = agg_les[0].keys()
+        today = timezone.now().date().isoformat()
+        filename = f"user_metrics-{today}.csv"
+        with open(filename, "w") as csvfile:
+            writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+            writer.writeheader()
+            writer.writerows(agg_les)
+        # TODO: save to S3?
```

Comment:
Did you want to include uploading this to S3?

Reply:
You're right, not hard to implement. I didn't see the point of convoluted env/cdk config when the name of the bucket is in the repo anyway, and the bucket doesn't need to be created or accessed programatically (just a human generating a report). For now.